pkgs/by-name/ss/sse2neon/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  stdenv,
```

Comment:
I've seen a preference for formatting package attributes with a leading comma:

```suggestion
{
, lib
, fetchFromGitHub
, pkg-config
, stdenv
```

Reply:
please see above